sdk/generic-sdk-core/generic-core/src/main/java/com/generic/core/http/client/DefaultHttpClient.java

```diff
@@ -151,10 +156,9 @@ private HttpURLConnection connect(HttpRequest httpRequest) {
      * Synchronously sends the content of an HttpRequest via an HttpUrlConnection instance.
      *
      * @param httpRequest The HTTP Request being sent
-     * @param progressReporter A reporter for the progress of the request
      * @param connection The HttpURLConnection that is being sent to
      */
-    private void sendBody(HttpRequest httpRequest, Object progressReporter, HttpURLConnection connection) {
+    private void sendBody(HttpRequest httpRequest, HttpURLConnection connection) {
```

Comment:
Do we not need `progressReporter` anymore?

Reply:
It wasn't used anywhere in the client. 
Was it purposeful only in async contexts?

Reply:
`ProgressReporter` was a concept added to support Storage scenarios where the customer can define a callback that updates as data is sent over the wire. But since this is internal implementation and we don't even have a `ProgressReporter` type in generic core this can be removed for now